EnumSoundID.h

```diff
@@ -227,7 +227,7 @@ enum SoundID
 	sndSavnt264		= 214, 	// 0xD6 "Ion drive module deployed"
 	sndSavnt265		= 215, 	// 0xD7 "Fusion drive module deployed"
 	sndSavnt266		= 216, 	// 0xD8 "Command module deployed"
-	sndSavnt267		= 217, 	// 0xD9 "Fuelling systems deployed"
+	sndSavnt267		= 217, 	// 0xD9 "Fueling systems deployed"

```

Comment:
Heh, according to Google, it seems this word has both spellings. The single "l" form is the American variant. A quick grep shows the game uses your corrected version though, and not what was originally used here.